app/components/mandatarissen/update-state.hbs

```diff
@@ -78,15 +78,13 @@
         </div>
       {{/if}}
       <div>
-        <AuLabel @required={{true}}>
-          Fractie
-        </AuLabel>
         {{#if this.load.isRunning}}
           <AuLabel for="input" class="skeleton-label au-u-margin-top" />
           <p class="skeleton-input-full au-u-margin-top au-u-margin-bottom"></p>
         {{else}}
           <Mandatarissen::FractieSelector
             @fractie={{this.selectedFractie}}
+            @isRequired={{this.isFractieSelectorRequired}}
             @onSelect={{this.updateFractie}}
             @allowClear={{false}}
```

Comment:
This allowClear is not used anymore.

Reply:
Will update it so it reacts on the is fractie required